src/linalg.jl

```diff
@@ -1,40 +1,59 @@
-export opInverse, opCholesky, opLDL, opHouseholder, opHermitian
+export opInverse, opCholesky, opLDL, opHouseholder, opHermitian 
+
+function mulFact!(res, F, v, α, β :: T) where T
+  if β == zero(T)
+    res .= α .* (F \ v)
+  else
+    res .= α .* (F \ v) .+ β .* res
```

Comment:
Can't we use `ldiv!()` here?

Reply:
We could, but we would have to compute the factorization explicitly inside the operator function. I think I just reproduced what was previously implemented here, we would change this but I would save it for another PR.